### DISCUSSION TITLE
Summary output

### DISCUSSION THREAD

**Discussion by Zeckie** [KotlinIsland/basedmypy#185] [Ideas]
Currently output ends with a summary like:
```Success: no issues found in 321 source files```
or
```Found 5 errors in 4 files (checked 321 source files)```

Would it be useful to include some baseline specific details, such as
- How many errors were ignored due to the baseline
- How many errors are in the baseline, but no longer in the code
- How many different types of error (in the baseline / code)

**Comment by KotlinIsland** [comment]
> How many errors are in the baseline, but no longer in the code

This should mostly be covered by auto-baseline, which feels like the most relevant to a normal mypy invokation. The other suggestions feel like they would fit better in some baseline specific commands, like `--baseline-report` or something.

Take a look at #93

**Comment by Zeckie** [comment]
> How many errors were ignored due to the baseline

I was thinking of something like:

`Found 5 errors in 4 files (checked 321 source files). Ignored 1234 errors in 54 files due to baseline file.`
or 
`Success: no issues found in 321 source files. Ignored 1234 errors in 54 files due to baseline file.`
